server/service/src/invoice/outbound_shipment/update/generate.rs

```diff
@@ -70,6 +83,42 @@ pub fn should_update_batches(
     }
 }
 
+// If status changed to allocated and above, remove unallocated lines
+fn unallocated_lines_to_trim(
+    connection: &StorageConnection,
+    invoice: &InvoiceRow,
+    status: &Option<UpdateOutboundShipmentStatus>,
+) -> Result<Option<Vec<InvoiceLineRow>>, RepositoryError> {
+    if invoice.status != InvoiceRowStatus::New {
```

Comment:
nit: comment that there is not status between New and Allocated?

Reply:
thanks: 708735921a70b25c4750a6d23ca27ab4e85d8ee5